src/chat-bubble/chat-bubble-passanger.ts

```diff
@@ -0,0 +1,77 @@
+import "../avatar";
```

Comment:
I think there's a typo in the name of this file

Reply:
Oopsie! 🤪

Reply:
Fixed ✅